utils/discord.js

```diff
@@ -39,8 +47,8 @@ const formatMessage = async ({ qty, card, totalPrice, buyer, seller, ethPrice, t
 					name: 'Curio Cards',
 					icon_url: 'https://www.gitbook.com/cdn-cgi/image/width=40,height=40,fit=contain,dpr=1,format=auto/https%3A%2F%2F1770801706-files.gitbook.io%2F~%2Ffiles%2Fv0%2Fb%2Fgitbook-x-prod.appspot.com%2Fo%2Fcollections%252FNyUEXr2B4FBefalwZTqb%252Ficon%252FAbV65JUMTbXuuBMe9I0M%252Favatar2.png%3Falt%3Dmedia%26token%3D99ceb163-5bff-440c-a5b4-83ac5ffc4d1a'
 				},
-				title: `Curio ${card} has been sold`,
-				description: `Platform: **${platform}**\nBuyer: **${buyerUsername}**\nSeller: **${sellerUsername}**\n---------------------------------`,
+				title: title,
+				description: `Platform(s): **${platforms.join(", ")}**\nBuyer: **${buyerUsername}**\nSeller: **${sellerUsername}**\n---------------------------------`,
```

Comment:
nitpick- can you change platform(s) to something like
`${platform.length > 1 ? "platforms" : "platform"`

Reply:
Done!